File: lab1b.c
#include <stdlib.h>
#include <stdio.h>

int main(){

    float a, b, c;

    printf("Digite o valor do primeiro lado: ");
    scanf("%f", &a);

    printf("Digite o valor do segundo lado: ");
    scanf("%f", &b);

    printf("Digite o valor do terceiro lado: ");
    scanf("%f", &c);

    if( a < b+c && b < a+c && c < a+b){

        if( a == b && a== c){

            printf("Triangulo Equilatero\n");

        } else if( a != b && a!= c && b != c){

            printf("Triangulo Escaleno\n");

        } else {

            printf("Triangulo Isoceles\n");

        }

    } else {

        printf("Nao existe triangulo com esses valores.\n");

    }

    system("pause");

    return 0;

}

File: lab1b.py
a = float(input("Primeiro segmento:"))
b = float(input("Segundo segmento:"))
c = float(input("Terceiro segmento:"))

if a <  b + c  and b <  a + c  and c <  a + b  :
    print("Os segmentos podem formar um triangulo ", end = "")
    if a == b and b == c :
        print("equilatero")
    elif a != b != c != a :
        print("escaleno")
    else:
        print("isóceles")
else :
    print("Triangulo nao existe")
    
    
        